tgui/packages/tgui/interfaces/Safe.js

```diff
@@ -119,20 +119,20 @@ const Help = (properties) => {
   return (
     <Section
       className="Safe--help"
-      title="Safe opening instructions (because you all keep forgetting)"
+      title="Инструкция по открытию сейфа."
```

Comment:
Забавный моментик вырезал из игры про шутки.
```suggestion
      title="Инструкция по открытию сейфа. (потому что вы всё время забываете)"
```